Script.php

```diff
@@ -737,136 +716,142 @@ function checksize($file)
 //}
 //   
 
-  $count=0;           
-           
-  
-  
+$count=0;           
+        
 
-  //check zise
-    if(!checksize("attachment1"))
-  {
-      echo "20 MB is the maximum file size allowed";
```

Comment:
**20MB** is too big.  Use **2MB**.   Why?  Our server is low on space.  

It would be great if we could configure this limit somewhere.  No a high priority now.


**Hui**

Reply:
We didnt affect these lines too but as i just checked i discovered thats a mistake it was supposed to be 2MB as according to this line below:

$result=$_FILES["$file"]['size']/1024/1024;

1024kb is around 1MB